### ISSUE TITLE
PDF files opening as text in new tab

### ISSUE THREAD

**Issue by rodonn** [andrewrynhard/atom-latex-plus#24]
Every time I compile the pdf of my tex file, Atom opens up a new tab with the contents of the pdf displayed as text (i.e. it looks mostly like gibberish).

Is there a way to get Atom either:
a) Not to open the pdf
b) Display the pdf in the new tab as a pdf

Thanks!

My json

```
{
  "projectPath": "~/",
  "title": "writeup",
  "root": "writeup.tex",
  "program": "pdflatex",
  "output": ".latex"
}
```

**Comment by rodonn** [comment]
I found the PDF view package. This makes allows Atom to at least display the pdf file correctly when the new tab opens up.

I'd still like to have an option that lets me compile the latex file without opening the resulting pdf. Is there a way to do that?

**Comment by andrewrynhard** [comment]
@rodonn the option to enable/disable automatically opening the PDF was merged with #25.
